core/src/main/scala/kafka/server/KafkaApis.scala

```diff
@@ -3743,8 +3743,48 @@ class KafkaApis(val requestChannel: RequestChannel,
   }
 
   def handleConsumerGroupDescribe(request: RequestChannel.Request): CompletableFuture[Unit] = {
-    requestHelper.sendMaybeThrottle(request, request.body[ConsumerGroupDescribeRequest].getErrorResponse(Errors.UNSUPPORTED_VERSION.exception))
-    CompletableFuture.completedFuture[Unit](())
+    val consumerGroupDescribeRequest = request.body[ConsumerGroupDescribeRequest]
```

Comment:
I just realized that we don't handle the `IncludeAuthorizedOperations` case in the request. When set, we must populate the `AuthorizedOperations` field in the response. You can check how we did this in `handleDescribeGroupsRequest`.

I suggest to do this in a separate pull request though.